api/descriptions/policies/vulnerability_host_get.md

```diff
@@ -0,0 +1,16 @@
+Retrieves the vulnerability policy for your host deployments.
```

Comment:
@KICKdesigns Let's use this sentence instead:

Retrieves the vulnerability policy for your hosts protected by Defender.